### ISSUE TITLE
Search input's aria label says Escape to cancel, but Escape has no effect

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#100927]
from @jvesouza:
When I press ctrl+shift+f orca reads the following: Search: Type Search Term and press Enter to search or Escape to cancel. However, the escape key has no effect after I press ctrl + shift + f

from me:
I recommend that we either update the aria label to not mention the Escape. Or to make escape actually clear / close search. I think the first approach is better.

**Comment by jvesouza** [comment]
How to clear / close the search?

**Comment by isidorn** [comment]
@jvesouza yeah, I guess the first escape should clear the search input and thus clearing the results.

So how to currently do that F1 > search clear resulsts, or shit + tab until you hit the search results button.
As for closing search: Ctrl + shift + F, or Ctrl + B

**Comment by roblourens** [comment]
Esc should cancel the running search, is there a different way we should announce that?

**Comment by israelpgjm** [comment]
> Esc should cancel the running search, is there a different way we should announce that?

**Comment by israelpgjm** [comment]
> Esc should cancel the running search, is there a different way we should announce that?

**Comment by israelpgjm** [comment]
> from @jvesouza:
> 
> When I press ctrl+shift+f orca reads the following: Search: Type Search Term and press Enter to search or Escape to cancel. However, the escape key has no effect after I press ctrl + shift + f
> 
> 
> 
> from me:
> 
> I recommend that we either update the aria label to not mention the Escape. Or to make escape actually clear / close search. I think the first approach is better.

**Comment by israelpgjm** [comment]
> @jvesouza yeah, I guess the first escape should clear the search input and thus clearing the results.
> 
> 
> 
> So how to currently do that F1 > search clear resulsts, or shit + tab until you hit the search results button.
> 
> As for closing search: Ctrl + shift + F, or Ctrl + B

**Comment by israelpgjm** [comment]
> Esc should cancel the running search, is there a different way we should announce that?

**Comment by israelpgjm** [comment]
> Esc should cancel the running search, is there a different way we should announce that?

**Comment by israelpgjm** [comment]
> @jvesouza yeah, I guess the first escape should clear the search input and thus clearing the results.
> 
> 
> 
> So how to currently do that F1 > search clear resulsts, or shit + tab until you hit the search results button.
> 
> As for closing search: Ctrl + shift + F, or Ctrl + B

**Comment by israelpgjm** [comment]
> from @jvesouza:
> 
> When I press ctrl+shift+f orca reads the following: Search: Type Search Term and press Enter to search or Escape to cancel. However, the escape key has no effect after I press ctrl + shift + f
> 
> 
> 
> from me:
> 
> I recommend that we either update the aria label to not mention the Escape. Or to make escape actually clear / close search. I think the first approach is better.

**Comment by isidorn** [comment]
@roblourens I think the following behavior would be idea:
search is in progress: escape stops the search in progress
search is not in progress: escape clears the search input

I personaly find this proposal intuitive.

**Comment by roblourens** [comment]
I disagree, if I double tab esc on accident then I will lose some info. If I press esc just as search is ending anyway, then I lose info. I don't think it's too hard to clear the search input already

**Comment by JacksonKearl** [comment]
We don't let Esc clear content in any of the other views, going to just remove the directive from the aria label.
